package.json

```diff
@@ -26,17 +29,17 @@
     "qs": "^6.10.3"
   },
   "devDependencies": {
-    "@nuxt/schema": "npm:@nuxt/schema-edge@latest",
+    "@nuxt/module-builder": "^0.1.6",
     "@nuxtjs/eslint-config-typescript": "^8.0.0",
     "@types/qs": "^6.9.7",
     "eslint": "^8.7.0",
     "nuxt3": "latest",
-    "ohmyfetch": "^0.4.14",
-    "standard-version": "^9.3.2",
-    "typescript": "^4.5.4",
-    "unbuild": "^0.6.7"
+    "standard-version": "^9.3.2"
   },
   "publishConfig": {
     "access": "public"
+  },
+  "build": {
+    "externals": ["vue", "nuxt3"]
```

Comment:
This has to be added to module-builder I guess (for future reference: I would gladly appreciate if you ping or create a small issue when using workarounds for build config)

Reply:
Fixed in @nuxt/module-builder@0.1.7

Reply:
Sure, I will create an issue with all the errors I encountered without this.

Reply:
https://github.com/nuxt/module-builder/issues/19